src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java

```diff
@@ -1,30 +1,60 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Classification;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.ScannableItem;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.isNull;
+
 public class EnvelopeMsg implements Msg {
 
+    @JsonProperty("id")
     private final String envelopeId;
 
-    private final byte[] envelope;
+    @JsonProperty("case_ref")
+    private String caseNumber;
 
-    private final boolean testOnly;
+    @JsonProperty("jurisdiction")
+    private String jurisdiction;
 
+    @JsonProperty("classification")
+    private Classification classification;
 
-    public EnvelopeMsg(String envelopeId) {
-        this(envelopeId, false);
-    }
+    @JsonProperty("doc_urls")
+    private List<String> documentUrls;
 
-    public EnvelopeMsg(String envelopeId, boolean testOnly) {
-        this.envelopeId = envelopeId;
+    @JsonIgnore
+    private final byte[] envelope;
```

Comment:
Do we need this field ? can we not just set all the required fields from envelope object to envelope msg model object ?

Reply:
No, we don't need it anymore. I'll construct the queue message body in getMsgBody() as per Luigi's comment.